api/HTMLEmbedElement.json

```
@@ -238,7 +238,7 @@
           "status": {
             "experimental": false,
             "standard_track": true,
-            "deprecated": true
+            "deprecated": false
```

The IDL for this has the `// also has obsolete members` comment link too.

```suggestion
            "deprecated": true
```